README.md

```diff
@@ -83,57 +83,38 @@ Full runnable examples can be found in `examples/` (e.g., [./examples/identity_a
 
 ## Building beman.exemplar
 
-### Dependencies
-<!-- TODO Darius: rewrite section!-->
-
-This project has no C or C++ dependencies.
-
-Build-time dependencies:
+### Build Environment
 
-- `cmake`
-- `ninja`, `make`, or another CMake-supported build system
-  - CMake defaults to "Unix Makefiles" on POSIX systems
-
-#### How to install dependencies
-
-<!-- TODO Darius: rewrite section!-->
+This project requires minimal **C++17** and **CMake 3.25** to build.
 
-<details>
-<summary>Dependencies install exemplar on Ubuntu 24.04  </summary>
-
-<!-- TODO Darius: rewrite section!-->
-
-```shell
-# Install tools:
-apt-get install -y cmake make ninja-build
+However,
+some examples and tests will not be compiled
+unless provided compiler support **C++20**.
```

Comment:
This should be put after "This project pull Google Test", and maybe highlight it with markdown's note functionality.

Reply:
Note that what is supported/ not supported can be seen in the CMake config log.

Reply:
done